### DISCUSSION TITLE
Posicionamento de campos em um form usando CSS Grid e Flexbox

### DISCUSSION THREAD

**Discussion by wilcorrea** [frontendbr/forum#1246] [Perguntas e Respostas]
Pessoal, estava querendo fazer um "rowspan" usando flexbox (isso mesmo, rs).

Eu consigo fazer com CSS Grid Layout, mas com Flexbox com o mesmo [markup desse pen ](https://codepen.io/wilcorrea/pen/gBBmyQ) estou dando como uma tarefa impossível.

A ideia é entregar o mesmo markup para CSS Grid e Flexbox. No momento as classes que fazem o efeito do rowspan só funcionam no CSS Grid : )

Alguém consegue me dar uma dica?

**Comment by tarciozemel** [comment]
Se eu entendi bem, você conseguiria fazer isso manipulando os valores de [`flex-basis`](http://desenvolvimentoparaweb.com/css/flexbox/#section_flex-basis)/[`flex-grow`](http://desenvolvimentoparaweb.com/css/flexbox/#section_flex-grow).

**Comment by mutheusalmeida** [comment]
Dá uma olhada:

https://codepen.io/matheusalmeida/pen/OaJWxR

https://codepen.io/matheusalmeida/pen/KxLYEO
